src/index.js

```diff
@@ -123,6 +123,15 @@ SteemConnect.prototype.comment = function comment(
   return this.broadcast([['comment', params]], cb);
 };
 
+SteemConnect.prototype.deleteComment = function deleteComment(author, permlink, cb) {
+  const params = {
+    author,
+    permlink
+  };
+  return this.broadcast([['delete_comment', params]], cb);
+};
+
+
```

Comment:
Here there is 2 empty lines, there should be only 1 line as separation between methods

Reply:
Fixed.